src/PrestaShopBundle/Command/ModuleCommand.php

```diff
@@ -83,7 +83,7 @@ protected function init(InputInterface $input, OutputInterface $output)
         $this->translator = $this->getContainer()->get('translator');
         $this->input = $input;
         $this->output = $output;
-        require $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
+        require_once $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
```

Comment:
Is this line still needed? :thinking:

Reply:
You're right, probably not !

Reply:
I remove all none necessary require config.inc.php of all default command